manifests/b/BandicamCompany/Bandicam/5.3.3.1893/BandicamCompany.Bandicam.installer.yaml

```diff
@@ -0,0 +1,21 @@
+# Created using wingetcreate 0.4.3.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: BandicamCompany.Bandicam
+PackageVersion: 5.3.3.1893
+MinimumOSVersion: 10.0.0.0
+InstallModes:
+- interactive
+- silent
+- silentWithProgress
+Installers:
+- InstallerLocale: en-US
+  Architecture: x86

```

Comment:
lets try this
```suggestion
  Architecture: x64
```
![image](https://user-images.githubusercontent.com/15158490/145285138-1c7da915-2350-4c7b-a9de-b5905226b8bf.png)